forge-gui-desktop/src/main/java/forge/itemmanager/filters/AdvancedSearchFilter.java

```diff
@@ -17,9 +17,9 @@
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.Iterables;
 
+import forge.frontend.components.widgets.IButton;
```

Comment:
I'm going to run a release this weekend for MOM. Will read through this after that.

Reply:
Cool!
Just let me know if you need anything.

Reply:
Not sure why this conversation was marked as resolved as I didn't read through this PR yet.